aws/scripts/py/deploy_content.py

```
@@ -29,7 +27,7 @@ def fetch_distributions():
 def get_staging_distribution(distributions):
     print("Finding staging distribution...")
     for item in distributions["DistributionList"]["Items"]:
-        if item["Staging"] == True:
+        if item["Staging"]:
             print(f"Staging distribution found: {item}")
             return item
```

Ruff

Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)